examples/best_practices/policy_validate_container_disallow_priviledgedprivelegesecalation.yaml

```diff
@@ -16,13 +16,12 @@ spec:
         - Pod
     validate:
       message: "Privileged mode is not allowed. Set allowPrivilegeEscalatin and privileged to false"
```

Comment:
For the message "Set allowPrivilegeEscalatin and privileged to false", the anyPattern rule checks for multiple conditions( allowPriveledgeEscalation: false + privileged: false & security Context for runAsNonRoot).
Suggested Message:
"Privileged mode is not allowed. Set allowPrivilegeEscalation and privileged to false or set runAsNonRoot: true"

Reply:
This policy does not validate runAsNonRoot. It checks allowPriveledgeEscalation and privileged both at pod level and container level.